### PR TITLE
common.mk: make CFG_IN_TREE_EARLY_TAS configurable at build time

### PR DESCRIPTION
**Pull request by jforissier**
Currently the list of early TAs cannot be overriden at build time, which is inconvenient. Say for instance one wants to build OP-TEE with only the PKCS#11 TA embedded, and not the trusted keys TA (or in addition to it). That would be:

make CFG_IN_TREE_EARLY_TAS=pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee

...or:

make CFG_IN_TREE_EARLY_TAS="pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee \
        trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c"

Update the OP-TEE OS build command in order to make the above work.

### CODE REVIEW

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: common.mk [hunk 1]
```diff
@@ -566,7 +568,7 @@ OPTEE_OS_COMMON_FLAGS ?= \
 	CFG_TEE_CORE_LOG_LEVEL=$(CFG_TEE_CORE_LOG_LEVEL) \
 	DEBUG=$(DEBUG) \
 	CFG_TEE_BENCHMARK=$(CFG_TEE_BENCHMARK) \
-	CFG_IN_TREE_EARLY_TAS=trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c
+	CFG_IN_TREE_EARLY_TAS=$(CFG_IN_TREE_EARLY_TAS)
```
Comment (etienne-lms) [inline common.mk:571]: I have to escape the space char separator in my shell command line to get to build to work.
Without seperartor:
```shell
$ make CFG_IN_TREE_EARLY_TAS="pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c"
make[1]: *** No rule to make target 'trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c'.
```
With the escape:
```shell
$ make CFG_IN_TREE_EARLY_TAS="pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee\ trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c"
... (runs ok)
```

Using quotes (simple or double) at line 571 seems to address the issue:
```patch
-	CFG_IN_TREE_EARLY_TAS=$(CFG_IN_TREE_EARLY_TAS)
+	CFG_IN_TREE_EARLY_TAS="$(CFG_IN_TREE_EARLY_TAS)"
```

Comment (jforissier) [reply common.mk]: Good catch! It shows that I did not test the very example that I gave :open_mouth:

Updated.

**Comment by jforissier** [comment]
Ping?

**Review by etienne-lms — COMMENTED** [review]
`Reviewed-by: Etienne Carriere <etienne.carriere@foss.st.com>`
`Tested-by: Etienne Carriere <etienne.carriere@foss.st.com>`

**Comment by etienne-lms** [comment]
Sorry @jforissier, I forgot to update that P-R. I've check the implementation quite a while ago...

**Pull request merged**